resources/views/backend/city/create.blade.php

```diff
@@ -0,0 +1,40 @@
+@extends('backend.layouts.master')
+@section('title', trans('labels.city'))
+@section('content')
+
+<!-- page content --> 
+
+<div class="col-md-12 col-sm-12 col-xs-12">
+	<div class="x_panel">
+		<div class="x_title">
+			<h2>{!! trans('labels.city') !!} <small>{!! trans('labels.create') !!}</small></h2>
+			<div class="clearfix"></div>
+			
+		</div>
+		<div class="x_content">
+			<br />
+                        
+                                {!! Form::open(array('route' => 'admin.city.index', 'id' => 'my-form', 'class' => 'form-horizontal form-label-left') ) !!}
```

Comment:
`'id' => 'my-form'` this id name is not good, please make the name more carefully. Remember naming is very importance